src/guidance/index.md.njk

```diff
@@ -0,0 +1,10 @@
+---
+title: Guidance
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+layout: layout-pane.njk
+show_subnav: true
+---
+
+Here you can find guidance to help you with your design process in HM Land Registry.
+
+The guidance has been created to consider HM Land Registry's ways of working and created alongside the involved parties.
```

Comment:
..created **alongside** the **involved** parties.

doesn't read well to me.

could we say - created with the subject matter experts

not being a CD - i'm happy to be ignored/overruled.

Reply:
Amend made. Similarly, happy for CD to overrule.